core/objects.go

```diff
@@ -239,6 +264,21 @@ type Challenge struct {
 	AccountKey *jose.JsonWebKey `json:"accountKey,omitempty"`
 }
 
+// UnsafeSetToken sets the token value both in the Token field and in the
+// serialized AuthorizedKey object.
+//
+// This method should only be used for writing tests, in cases where the
```

Comment:
I poked around a little to see if there was something we could do to try and catch use of this method in prod code but I don't see much other than conditional compilation.

Anyway, I'm sure it's fine. Just was curious.